README.md

```diff
@@ -17,12 +17,14 @@ I) Code quality
 - [ ] Easily embeddable (*and take part of the standardisation of the "C/C++ embedding API"*)
 
 II) Simplicity
+- [x] Only implement WebAssembly 1.0 (and none of the proposals)
 - [x] Interpreter only
-- [x] Provide no support for floating point operations (initially)
 - [x] Support only WebAssembly binary encoding as an input (no support for the text format (`.wat`/`.wast`))
 
 III) Conformance
-- [ ] Should pass the official WebAssembly test suite
+- [x] Should have 99+% unit test coverage
```

Comment:
I'd say we aim for 100% although it may be difficult to get precisely that because of the toolchain bugs and limitations.

Reply:
That's why I put 99%, but I'm fine either way.

Reply:
"99+%" looks good to me